src/main/java/org/broadinstitute/hellbender/tools/exome/GetBayesianHetCoverage.java

```diff
@@ -24,35 +24,111 @@
+ * Calls heterozygous SNPs, counts ref and alt alleles and gives genotype likelihoods.
  *
- * The tool has three modes. The mode is automatically selected based on the provided arguments:
+ * <p>The tool has three modes. The mode is automatically selected based on the provided arguments as follows. </p>
  *
  * <dl>
- *     <dt>NORMAL_ONLY</dt><dd>This mode is chosen if just the normal BAM file is provided. The balanced prior
- *     is used for estimating the likelihood of het sites (see {@link BalancedHeterozygousPileupPriorModel}).</dd>
+ *      <dt>MATCHED_NORMAL_TUMOR</dt><dd>Mode when given both matched normal and tumor BAM files.
+ *      Estimates the likelihood of heterozygous genotypes on the normal reads using a balanced prior
+ *      (see {@link BalancedHeterozygousPileupPriorModel}). Counts bases for the same sites for the
+ *      tumor reads.</dd>
  *
- *      <dt>TUMOR_ONLY</dt><dd>This mode is chosen if just the tumor BAM file is provided. The heterogeneous prior
- *      is used for estimating the likelihood of het sites (see {@link HeterogeneousHeterozygousPileupPriorModel}).</dd>
+ *      <dt>TUMOR_ONLY</dt><dd>For instances lacking a matched normal.
+ *      Mode when given just the tumor BAM file with the --tumor argument. Estimates the likelihood of heterozygous genotypes
+ *      using the heterogeneous prior (see {@link HeterogeneousHeterozygousPileupPriorModel}).
+ *      This mode has only been tested for low purity tumors, i.e. with high normal contamination.
+ *      For high purity tumors, both sensitivity and precision may decrease.</dd>
  *
- *      <dt>MATCHED_NORMAL_TUMOR</dt><dd>This mode is chosen if both normal and tumor BAM files are provided.
- *      The balanced prior (see {@link BalancedHeterozygousPileupPriorModel}) is used to estimate the likelihood of
- *      Het sites on the normal reads. The base counts on the same het sites are collected and reported from the
- *      tumor reads.</dd>
+ *     <dt>NORMAL_ONLY</dt><dd>Extended use-case for germline heterozygous site coverage.
+ *     Mode when given just the normal BAM file with the --normal argument.
+ *     Estimates the likelihood of heterozygous genotypes using the balanced prior
+ *     (see {@link BalancedHeterozygousPileupPriorModel}).</dd>
  * </dl>
+ *
+ * <p>
+ *     The tool writes a table with the following columns:
+ *
+ *     <pre>
+ *     CONTIG,
+ *     POSITION,
+ *     REF_COUNT,
+ *     ALT_COUNT,
+ *     REF_NUCLEOTIDE,
+ *     ALT_NUCLEOTIDE,
+ *     READ_DEPTH,
+ *     HET_LOG_ODDS
+ *     READ_DEPTH = total pileup depth at the SNP site
+ *     HET_LOG_ODDS = log odds of the SNP site being heterozygous
+ *     </pre>
+ * </p>
+ *
+ * <p>
+ *     The required --snpIntervals file gives the genomic positions of common population Single Nucleotide Polymorphisms (SNPs)
+ *     in Picard-style interval list format. See https://broadinstitute.github.io/picard/command-line-overview.html#BedToIntervalList for details.
```

Comment:
You might want to put external links like this in anchor tags, so in the end result we;ll get real links instead of seeing the url:

<a href=https://broadinstitute.github.io/picard/command-line-overview.html#BedToIntervalList>BedToIntervalList</a>